### DISCUSSION TITLE
v2.1.3

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2975] [分享 / Share]
从本次更新起, 试行新的更新策略, 正式版会以更高频率接受预览版中功能修复类的改动 (`preview-fixes` 分支).

- 下载视频:
  - 修复部分番剧无法下载. (#2942)
  - 修复切换视频后信息未更新. (#2805)
  - 支持 2022 拜年祭. (#2965)
- 修复 `关于` 中的主页链接错误.
- 自定义顶栏:
  - 修复专栏中的顶栏没有自动隐藏.
  - 修复在历史记录页面中的样式. (#2961, #2962)
  - 修复选项出现 null 导致脚本无法运行. (#2928)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.3'>v2.1.3</a>.</em>

**Comment by 2SHANINGITHUB** [comment]
顶栏组件找不到更新，Commit 版本一直是f48fdb070，脚本已经更新到v2.1.3了

**Reply by the1812** [reply]
https://github.com/the1812/Bilibili-Evolved/issues/2928#issuecomment-1031192076

**Comment by Hukeqing** [comment]
想问下v1的那种清爽主页完成了吗，似乎没有找到类似的说明

**Reply by Geeyun-JY3** [reply]
https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.2
